code/_onclick/cyborg.dm

```diff
@@ -77,6 +77,7 @@
 		//If the gripper contains something, then we will use its contents to attack
 		if (G.wrapped && (G.wrapped.loc == G))
 			GripperClickOn(A, params, G)
+			G.update_icon() //We may need to update our gripper based on a change in the wrapped item
```

Comment:
wouldn't this be G.wrapped.update_icon() as well? or does the gripper's update_icon handle that

Reply:
GripperClickOn() handles that, because it calls wrapped.attack() and that already handles all required update_icon()s.